src/pages/commits/ActiveCommits/CommitChartTooltip.tsx

```diff
@@ -32,19 +32,24 @@ export const CommitChartTooltip: React.FC<Props> = ({
       <TooltipContainer data-cy="commit-chart-tooltip" css={sharedCss}>
         {groupedTaskStats.map(({ color, count, statuses, umbrellaStatus }) => (
           <FlexColumnContainer key={color} data-cy="current-status-count">
-            <TotalCount status={umbrellaStatus} color={color} count={count} />
+            <TotalCount
+              status={umbrellaStatus}
+              color={color}
+              count={count}
+              active
+            />
             {umbrellaStatus === TaskStatus.Failed && (
               <SubStatusText css={sharedCss}>{`(${statuses.join(
                 ", "
               )})`}</SubStatusText>
             )}
           </FlexColumnContainer>
         ))}
-        {zeroCountStatus.map((status) => (
+        {zeroCountStatus.map((umbrellaStatus) => (
           <TotalCount
-            key={status}
-            status={status}
-            color={mapTaskStatusToColor[status]}
+            key={umbrellaStatus}
+            status={umbrellaStatus}
+            color={mapUmbrellaStatusColors[umbrellaStatus].border}
```

Comment:
Nice catch. The color field actually maps to `barChart` instead of `border` https://github.com/evergreen-ci/spruce/pull/886/files#diff-fb5d09bb6b48aa77ae57c5ce3398f59dc7200b1c8b1ad3c2ede6161096198e28R31

We won't need to access `mapUmbrellaStatusColors` here once the taskStatusCounts resolver includes zero counts ([ticket](https://jira.mongodb.org/browse/EVG-15459))